roles/acm/setup_ksops/tasks/main.yml

```
@@ -0,0 +1,58 @@
+  kubernetes.core.k8s:
+    definition:
+      apiVersion: argoproj.io/v1beta1
+      kind: ArgoCD
+      metadata:
+        name: openshift-gitops
+        namespace: openshift-gitops
+      spec:
+        kustomizeBuildOptions: --enable-alpha-plugins --enable-exec
+        repo:
+          env:
+           - name: XDG_CONFIG_HOME
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 12 but found 11

(indentation)